package.json

```diff
@@ -35,49 +34,48 @@
   },
   "dependencies": {
     "ember-ajax": "^5.0.0",
-    "ember-auto-import": "^1.0.1",
-    "ember-cli-babel": "^7.7.3",
-    "ember-cli-release": "^1.0.0-beta.2",
+    "ember-auto-import": "^1.5.2",
+    "ember-cli-babel": "^7.12.0",
     "ember-get-config": "^0.2.4",
-    "ember-simple-auth": "^1.6.0",
+    "ember-simple-auth": "^1.9.2",
     "uuid": "^3.3.2"
   },
   "devDependencies": {
-    "@adfinis-sygroup/semantic-release-config": "^2.0.0",
-    "@ember/jquery": "^0.6.0",
-    "@ember/optional-features": "^0.7.0",
+    "@adfinis-sygroup/semantic-release-config": "^2.1.4",
+    "@babel/core": "^7.6.2",
+    "@ember/jquery": "^1.0.0",
+    "@ember/optional-features": "^1.0.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.10.0",
-    "ember-cli-dependency-checker": "^3.1.0",
-    "ember-cli-eslint": "^5.1.0",
-    "ember-cli-htmlbars": "^3.0.1",
-    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
+    "ember-cli": "~3.12.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^4.0.4",
     "ember-cli-inject-live-reload": "^2.0.1",
-    "ember-cli-mirage": "^0.4.7",
+    "ember-cli-mirage": "^1.1.2",
     "ember-cli-sri": "^2.1.1",
-    "ember-cli-template-lint": "^1.0.0-beta.1",
+    "ember-cli-template-lint": "^1.0.0-beta.3",
     "ember-cli-uglify": "^3.0.0",
     "ember-disable-prototype-extensions": "^1.1.3",
     "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^2.0.0",
+    "ember-load-initializers": "^2.1.0",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-qunit": "^4.4.1",
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.10.0",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.0.0",
-    "eslint-config-prettier": "^4.2.0",
-    "eslint-plugin-ember": "^6.2.0",
-    "eslint-plugin-node": "^9.0.1",
-    "eslint-plugin-prettier": "2.6.0",
-    "husky": "^2.3.0",
-    "lint-staged": "^8.1.7",
+    "ember-qunit": "^4.5.1",
+    "ember-resolver": "^5.3.0",
+    "ember-source": "~3.12.0",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-try": "^1.2.1",
+    "eslint": "^6.5.1",
+    "eslint-config-prettier": "^6.3.0",
+    "eslint-plugin-ember": "7.1.0",
+    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-prettier": "3.1.1",
+    "husky": "^3.0.8",
+    "lint-staged": "^9.4.1",
     "loader.js": "^4.7.0",
-    "prettier": "^1.13.7",
-    "qunit-dom": "^0.8.4"
+    "prettier": "^1.18.2",
+    "qunit-dom": "^0.9.0"
```

Comment:
We should pin the dev dependencies to an exact version

Reply:
Done